ambry-network/src/main/java/com.github.ambry.network/NetworkMetrics.java

```diff
@@ -78,6 +79,26 @@
   // the count of renegotiation after initial handshake done
   public final Counter sslRenegotiationCount;
 
```

Comment:
1. To simply out, anything under network package are considered to be network layer. 
2. @pnarayanan @xiahome : I went over the code once again and realized that there was a bug. Basically the active connections for selector metric was never registered and hence we didn't hit the issue. I have fixing it along with network client's pending connections metric. Do take a look. I am just aggregating all the values from all (selector/networkclient) and update just one metric.
3. Yes, could be. Since this metric is un-related to this patch, can you create a new issue.